### DISCUSSION TITLE
Enabling surround sound

### DISCUSSION THREAD

**Discussion by MurasakiZerozaki** [open-goal/jak-project#3470] [Q&A] [answered]
I'm using surround sound as 4.0 via motherboard and it works perfectly in any games, music or mediaplayers, but Jak II (tried only this) still stereo. Yes, I've enabled this option in settings. Yes, already restarted game to check it. Is it still in work?

**Comment by ManDude** [comment, marked as answer]
It's in development.

**Reply by MurasakiZerozaki** [reply]
Ok, good. Will test later.
